context/app/static/js/components/searchPage/ResultsTable/utils.jsx

```diff
@@ -48,6 +51,14 @@ function getByPath(hitSource, field) {
   }
 
   if (Array.isArray(fieldValue)) {
+    if (field?.id === 'mapped_metadata.age_value') {
+      return (
+        <Stack direction="row">
+          {fieldValue.join(' / ')}
+          <DonorAgeTooltip donorAge={fieldValue.join(' / ')} />
```

Comment:
Will this be able to be converted with `Number`?

Reply:
Yes, because the `fieldValue` array only contains one element - but it might be a bit safer to take only the first element of `fieldValue`.